CHANGES.md

```diff
@@ -1,3 +1,10 @@
+# next -- TBA
+
+## Bug fixes
+
+* Counterexamples including arrays are now printed with the array
+  contents instead of placeholder text.
```

Comment:
Perhaps we should clarify what type of arrays we are talking about here:

```suggestion
* Counterexamples including SMT arrays are now printed with the array
  contents instead of placeholder text.
```

Otherwise, this might be confused with LLVM or MIR arrays.

Reply:
Good point. fixed in 11b0ba14638f55d7df33709db3077a84d3746085